docs/install.md

```diff
@@ -1,93 +1,58 @@
 <!--
 ---
-linkTitle: "Installation"
+linkTitle: "Setting Up Tekton Triggers"
 weight: 2
 ---
 -->
-# Installing Tekton Triggers
+# Setting Up Tekton Triggers
 
-Use this page to add the component to an existing Kubernetes cluster.
+Follow the steps below to set up an official release of Tekton Triggers on your Kubernetes cluster. 
+If you want to test Triggers from `HEAD`, see the
+[Tekton Developer Guide](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers).
 
-- [Pre-requisites](#pre-requisites)
-- [Versions](#versions)
-- [Installing Tekton Triggers](#installing-tekton-triggers-1)
+## Prerequisites
 
-## Pre-requisites
+You must satisfy the following prerequisties to set up Tekton Triggers:
 
-1.  A Kubernetes cluster version 1.18 or later
+* You must have Tekton Pipelines installed on your Kubernetes cluster. For instructions, see [Installing Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
 
-    If you don't already have a cluster, you can create one for testing with
-    `kind`.  [Install
```

Comment:
I think it is helpful to still briefly mention kind here in this context. imo, it helps ensure we have a path with low barriers to contribution and get folks on a common path.

Reply:
Restored the `kind` instructions as per review comments.